<h1>Numbers and Math</h1>


In all programs different types of functions and symbols are used to perform mathematical operations, so mathematical program that facilitates the person in solving problems in a matter of seconds without any problem.

Below are some basic mathematical functions to perform operations:

• + plus

• - minus

• / slash

• * asterisk

• % percent

• < less-than

• > greater-than

• <= less-than-equal

• >= greater-than-equal


Below some exercises:

In [19]:
print("I will now count my spicy sweets and sugary sweets:")

I will now count my spicy sweets and sugary sweets:


In [13]:
print("spicy sweets", 30 + 5 / 4)

spicy sweets 31.25


In [14]:
print("sugary sweets", 15 - 33 * 2 % 5)

sugary sweets 14


Others problems.

In [11]:
print("Is it true that 5 + 5 > 5 - 7?")

Is it true that 3 + 2 > 5 - 7?


In [12]:
print(5 + 5 > 5 - 7)

True


In [16]:
print("Is it greater?", 7 > 4)


Is it greater? True


In [17]:
print("Is it greater or equal?", 8 >= 6)


Is it greater or equal? True


In [18]:
print("Is it less or equal?", 12 <= 7)


Is it less or equal? False


I will now count my spicy sweets and sugary sweets:
spicy sweets 31.25
sugary sweets 14

Is it true that 5 + 5 > 5 - 7?
True
What is 3 + 2? 5
What is 5 - 7? -2

How about some more.
Is it greater? True
Is it greater or equal? True
Is it less or equal? False
